source/clear-linux/guides/maintenance/mixer.rst

```diff
@@ -22,6 +22,13 @@ add it with the :command:`swupd bundle-add` command as follows:
 
 Current mixing workflow
 ***********************
+Mixer by default runs *all* build commands in a container to ensure the
+correct version of the tooling is being used. This also allows custom mixes
+to automatically perform downstream format bumps when upstream releases
+a format bump. You can still run mixer natively by appending --native to
+any of the commands. Note that you cannot run mixer if you are already in a
```

Comment:
Elsewhere in this doc, notes are called out in a note box (example in step 10 under "Create ngninx web server .."). Consider calling out this note in the same manner, using the note directive.